migration/v1_to_v2/data/exporters/record_data_exporter.rb

```diff
@@ -17,6 +22,46 @@ def migrate_notes(notes)
     end
   end
 
+  def get_radio_fields
+    primero_module = PrimeroModule.find_by_name("CP")  #hardcoded for now. need to change it to accomodate either one. Cannot use all since there comflicting fields in both modules
```

Comment:
We'll need to change this to accommodate GBV migrations. There are shared fields between the two modules but they shouldn't conflict.